src/qt/guiutil.cpp

```diff
@@ -671,14 +671,26 @@ QString ConnectionTypeToQString(ConnectionType conn_type, bool prepend_direction
 {
     QString prefix;
     if (prepend_direction) {
-        prefix = (conn_type == ConnectionType::INBOUND) ? QObject::tr("Inbound") : QObject::tr("Outbound") + " ";
+        prefix = (conn_type == ConnectionType::INBOUND) ?
+                     /*: An inbound connection from a peer. An inbound connection
+                         is a connection initiated by a peer. */
+                     QObject::tr("Inbound") :
+                     /*: An outbound connection to a peer. An outbound connection
+                         is a connection initiated by us. */
+                     QObject::tr("Outbound") + " ";
     }
     switch (conn_type) {
     case ConnectionType::INBOUND: return prefix;
+    //: Peer connection type that relays all network information.
     case ConnectionType::OUTBOUND_FULL_RELAY: return prefix + QObject::tr("Full Relay");
+    /*: Peer connection type that relays relays network information about
```

Comment:
The updated translated comment looks good. But I think there is some scope for correction. This might be an intentional decision by OP. But I still better mention it.

```suggestion
    /*: Peer connection type that relays relay network information about
```
or

```suggestion
    /*: Peer connection type that relays network information about
```

Reply:
was not intentional :D, thanks for catching this. Will update

Reply:
thanks, addressed in 371e2b9